app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -12,30 +12,30 @@ import com.mapbox.vision.teaser.view.show
 import com.mapbox.vision.safety.VisionSafetyManager
 import com.mapbox.vision.teaser.MainActivity.VisionManagerMode.*
 import com.mapbox.vision.teaser.ar.ArMapActivity
+import com.mapbox.vision.teaser.recorder.RecorderFragment
 import com.mapbox.vision.teaser.replayer.ArReplayNavigationActivity
 import com.mapbox.vision.teaser.replayer.ReplayModeFragment
 import com.mapbox.vision.teaser.view.hide
 import com.mapbox.vision.view.VisionView
+import com.mapbox.vision.view.VisualizationMode
 import kotlinx.android.synthetic.main.activity_main.*
 
-class MainActivity : BaseTeaserActivity(), ReplayModeFragment.OnClickModeItemListener {
+class MainActivity : BaseTeaserActivity(), ReplayModeFragment.OnSelectModeItemListener {
 
     enum class VisionManagerMode {
```

Comment:
Rename Mode in this PR?

Reply:
Let's do in next PR, it will contain both modes together: AppMode, and VisionManagerMode, would be nice to discuss good naming together